codeguru/views.py

```diff
@@ -60,7 +60,13 @@ def group(request, id=None):
     if not current_group:
         return error(request, "Group not found.")
 
-    members = Profile.objects.all().filter(group=current_group)
+    member_profiles = Profile.objects.all().filter(group=current_group)
+    members = [user for user in User.objects.all() if (user.profile in member_profiles)]

```

Comment:
There is probably a way to get the user from the profile without filtering for all users. 

Maybe [this page](https://docs.djangoproject.com/en/5.1/topics/db/examples/one_to_one/) will help